src/wp-includes/class-wp-scripts.php

```diff
@@ -419,14 +471,15 @@ public function do_item( $handle, $group = false ) {
 	 *
 	 * @since 4.5.0
 	 *
-	 * @param string $handle   Name of the script to add the inline script to.
-	 *                         Must be lowercase.
-	 * @param string $data     String containing the JavaScript to be added.
-	 * @param string $position Optional. Whether to add the inline script
-	 *                         before the handle or after. Default 'after'.
+	 * @param string $handle     Name of the script to add the inline script to.
+	 *                           Must be lowercase.
+	 * @param string $data       String containing the JavaScript to be added.
+	 * @param string $position   Optional. Whether to add the inline script
+	 *                           before the handle or after. Default 'after'.
+	 * @param bool   $standalone Optional. Inline script opted to be standalone or not. Default false.
 	 * @return bool True on success, false on failure.
 	 */
-	public function add_inline_script( $handle, $data, $position = 'after' ) {
+	public function add_inline_script( $handle, $data, $position = 'after', $standalone = false ) {
```

Comment:
same question here - should we consider updating all usages in core?

Reply:
@adamsilverstein I think this would be okay to do, my only comment would be that _should_ any of the main scripts that the inline scripts attached to ever be updated to have a deferred loading strategy (even by association with another script), we'd need to be very certain that these `$standalone` declarations do not affect the inline scripts, although I am quite sure that should not be the case.